test/auth.test.js

```diff
@@ -1,16 +1,16 @@
 import sinon from 'sinon';
-import StitchClient from '../src/client';
+import { StitchClient } from '../src/client';
 import * as common from '../src/auth/common';
 import { PROVIDER_TYPE_ANON, PROVIDER_TYPE_USERPASS } from '../src/auth/providers';
 import StitchMongoFixture from './fixtures/stitch_mongo_fixture';
 import { buildClientTestHarness, extractTestFixtureDataPoints } from './testutil';
 
 function mockAuthData() {
   const data = {
-    accessToken: 'fake-access-token',
-    refreshToken: 'fake-refresh-token',
-    userId: 'fake-user-id',
-    deviceId: 'fake-device-id'
+    access_token: 'fake-access-token',
```

Comment:
[q] Why did this need to change?

Reply:
The test previously was directly write to storage with pre-massaged values. This probably wasn't correct in the first place, and using `storage` as it wasn't meant to be used isn't great in a test not even testing `storage`.

Reply:
👍